libkbfs/data_types.go

```diff
@@ -62,6 +62,15 @@ type TeamInfo struct {
 	// whether a user was a member given some Merkle seqno?
 }
 
+// ImplicitTeamInfo contains information needed after
+// resolving/identifying an implicit team.  TeamInfo is used for
+// anything else.
```

Comment:
Is this gonna be used for single teams as well, or is the plan to use the old code path for single teams since we won't need a resolve for them? If former, it sounds like this could use a different name.

Reply:
I think it won't be used for single teams.  Though once we start storing the TLF ID in the sigchain for regular teams, it probably makes sense to either add or move the TLF ID over to `TeamInfo`.

Note that the `Name` fields of the two structs are different.  For implicit teams, the `TeamInfo` will contain the auto-generated name, while the `ImplicitTeamInfo` contains the nice "display" name.

Reply:
Got you; sounds good. Thanks for clarifying that and adding comment!